pom.xml

```diff
@@ -223,6 +223,36 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>travis</id>
+            <activation>
+                <property>
+                    <name>travis</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <executions>
+                         <!-- default test execution -->
+                            <execution>
+                                <id>default-test</id>
+                                <phase>test</phase>
+                                <goals>
+                                 <goal>test</goal>
+                                </goals>
+                                <configuration>
+                                     <!--Temporarily exclude the tests only failed on Travis-->
+                                     <excludes>
+                                     </excludes>
+                                </configuration>

```

Comment:
I tried. After https://issues.jboss.org/browse/RESTEASY-2011,
we can remove the tests exclusion in testsuite/integration-tests/pom.xml.

Reply:
@jimma not sure I understand, sorry. What's the link with RESTEASY-2011 ? And why would we want to remove the tests exclusions in testsuite/integration-tests/pom.xml? The idea is that the exclusions here (related to / reproducible only with Travis env) would add to the exclusions that are already in place in the testsuite (because of un-finished tests or functionalities, known problems on old containers, etc)

Reply:
@asoldano  I meant we now adding combin.children="append" will take affect for this exclusion item in testsuite/integration-tests/pom.xml: https://github.com/resteasy/Resteasy/blob/master/testsuite/integration-tests/pom.xml#L483. After we fixed RESTEASY-2011, we can remove this exclusion.
If there is exclusion in child module, it can be moved in travis profile.

Reply:
@asoldano  Added in the newest commit.